src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -568,6 +570,31 @@ enum AppStrings {
 		static let hygieneRulesPoint4 = NSLocalizedString("ExposureDetection_Info_HygieneRules_Point4", comment: "")
 		static let hygieneRulesPoint5 = NSLocalizedString("ExposureDetection_Info_HygieneRules_Point5", comment: "")
 		static let hygieneRulesTitleImageDescription = NSLocalizedString("ExposureDetection_Info_HygieneRules_TitleImage_Description", comment: "")
+		
+		static let contagionTitle = NSLocalizedString("ExposureDetection_Info_Contagion_Title", comment: "")
```

Comment:
We don't have spaced in between the strings in AppStrings, could you please remove the space for consistency purpose?

Reply:
removed spaces.